netbox/extras/tables/tables.py

```diff
@@ -545,15 +546,30 @@ class ScriptResultsTable(BaseTable):
         template_code="""{% load log_levels %}{% log_level record.status %}""",
         verbose_name=_('Level')
     )
+    object = tables.Column(
+        verbose_name=_('Object')
+    )
+    url = tables.Column(
+        verbose_name=_('URL')
+    )
```

Comment:
I would ditch the URL column. IMO it's poor form to print a raw URL as a value, and it's redundant to the `object` column.